docs/source/index.rst

```diff
@@ -37,6 +37,7 @@ Documentation
 
    installation
    geo_def
+   spherical_geometry
```

Comment:
Please put this lower. Spherical geometry is not what most people are coming to pyresample for. I could see this going below `data_reduce` or maybe above `plot`. As is this is going to be the second item after geometry objects are described and before the utilities about how to make those geometries.

Reply:
I agree, it is a bit of a special functionality of Pyresample. I have put it above plots now. What do you think?

Reply:
Works for me. I think that's a good place as everything above is "use pyresample to do this stuff" and everything below is kind of utility functions and stuff.